packages/design-system/src/components/button.tsx

```diff
@@ -96,10 +96,11 @@ const perColorStyle = (variant: ButtonColor) => ({
           ),
   },
 
-  "&:disabled:not([data-state=pending]), &[data-state=disabled]": {
-    background: theme.colors.backgroundButtonDisabled,
-    color: theme.colors.foregroundDisabled,
-  },
+  "&:disabled:not([data-state=pending]), &[data-state=disabled], &[data-state=disabled]:visited, &[data-state=disabled]:hover":
```

Comment:
same there is perfectly valid aria-disabled=true

Reply:
Moved